packages/wdio-shared-store-service/src/server.ts

```diff
@@ -46,6 +51,57 @@ export const startServer = () => new Promise<{ port: number, app: PolkaInstance
             store[key] = req.body.value as JsonCompatible | JsonPrimitive
             return res.end()
         })
+        .post('/setResourcePool', (req, res, next) => {
+            const key = req.body.key as string
+            const value = req.body.value as JsonCompatible | JsonPrimitive
+
+            if (!Array.isArray(value)) {
+                next(new Error('Resource pool must be an array of values'))
+                return
+            }
+
+            resourcePoolStore.set(key, value)
+            return res.end()
+        })
+        .post('/getValueFromPool/:key', async (req, res, next) => {
```

Comment:
@christian-bromann setting this to `.get` caused me issues, unit tests that throw errors would hang for 2s instead of an immediate reply. `.post()` throws the error immediately

Reply:
Let's fix the unit test then, this should definitely be a get method. Given it is not really a user facing API I am happy to ignore the url path.

Reply:
figured it out, `got` has a retry mechanism on error, I've set all `throwHttpErrors: false, retry: { limit: 0 }`

Reply:
also, to note: `GET` operations should not modify the state of data, that's why got was retrying, assuming that it wouldn't make a difference in the server. A `POST` operation is more suited to our api call because it's actually modifying internal data